source/utilities.tex

```diff
@@ -14997,20 +14997,16 @@
 The extended grapheme clusters of a string are defined by \UAX{29}.
 The estimated width of the following code points is 2:
 \begin{itemize}
-\item \ucode{1100} -- \ucode{115f}
-\item \ucode{2329} -- \ucode{232a}
-\item \ucode{2e80} -- \ucode{303e}
-\item \ucode{3040} -- \ucode{a4cf}
-\item \ucode{ac00} -- \ucode{d7a3}
-\item \ucode{f900} -- \ucode{faff}
-\item \ucode{fe10} -- \ucode{fe19}
-\item \ucode{fe30} -- \ucode{fe6f}
-\item \ucode{ff00} -- \ucode{ff60}
-\item \ucode{ffe0} -- \ucode{ffe6}
-\item \ucode{1f300} -- \ucode{1f64f}
-\item \ucode{1f900} -- \ucode{1f9ff}
-\item \ucode{20000} -- \ucode{2fffd}
-\item \ucode{30000} -- \ucode{3fffd}
+\item
+Any code point with the East_Asian_Width="W" or
+East_Asian_Width="F" Derived Extracted Property as described by
```

Comment:
I think these should be formatted with code font.
```
$ git grep -C 1 'Unicode property' *.tex
utilities.tex-\placeholder{C} corresponds to either
utilities.tex:a UCS scalar value whose Unicode property \tcode{General_Category}
utilities.tex-has a value in the groups \tcode{Separator} (\tcode{Z}) or \tcode{Other} (\tcode{C}) or to
utilities.tex:a UCS scalar value which has the Unicode property \tcode{Grapheme_Extend=Yes},
utilities.tex-as described by table 12 of \UAX{44}, or
```

Reply:
I \tcode'd the `East_Asian_Width` assignments, but what about "Derived Extracted Property"?

Reply:
Seems fine as is.